kubeflow-pipelines/docling-standard/standard_convert_pipeline.py

```diff
@@ -88,6 +98,21 @@ def convert_pipeline(
         converter.set_cpu_request("500m")
         converter.set_cpu_limit("4")
 
+        # Step 5: Optionally chunk the converted documents
+        # When docling_chunk_enabled=True, chunk the Docling JSON output
+        # Output files will be saved as {filename}_chunks.json
+        with dsl.If(docling_chunk_enabled == True):  # noqa: E712
+            chunker = docling_chunk(
+                input_path=converter.outputs["output_path"],
+                max_tokens=docling_chunk_max_tokens,
+                merge_peers=docling_chunk_merge_peers,
+            )
+            chunker.set_caching_options(False)
```

Comment:
Why set these values here after `docling_chunk()` has already been called? What's the point of returning a `chunker` from `docling_chunk()` at all?